src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -7,7 +7,7 @@
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person name provided is invalid";
```

Comment:
Since we are using `Patient` now, maybe we can rename some of these variables? 

Perhaps from `MESSAGE_INVALID_PERSON_DISPLAYED_INDEX` to `MESSAGE_INVALID_PATIENT_DISPLAYED_NAME`.

Also, do you think we should change our usage of "person" to "patient"? 🤔

 i.e. "The **patient** name..." instead.

Reply:
okay, let me change it now and update the pr